build_runner/lib/src/build_script_generate/build_script_generate.dart

```diff
@@ -50,8 +50,14 @@ Future<Iterable<Expression>> _findBuilderApplications(String configKey) async {
     if (buildConfigOverrides.containsKey(package.name)) {
       return buildConfigOverrides[package.name];
     }
-    return BuildConfig.fromBuildConfigDir(
-        package.name, package.dependencies.map((n) => n.name), package.path);
+    try {
+      return await BuildConfig.fromBuildConfigDir(
+          package.name, package.dependencies.map((n) => n.name), package.path);
+    } catch (e) {
+      // During the build an error will be logged
```

Comment:
[nit] end comments with a period (here and below): https://www.dartlang.org/guides/language/effective-dart/documentation#do-format-comments-like-sentences

Reply:
Done